plotly/plotly/plotly.py

```diff
@@ -869,7 +868,7 @@ def _fill_in_response_column_ids(cls, request_columns,
         for req_col in request_columns:
             for resp_col in response_columns:
                 if resp_col['name'] == req_col.name:
-                    req_col.id = '{0}/{1}'.format(grid_id, resp_col['uid'])
+                    req_col.id = '{0}:{1}'.format(grid_id, resp_col['uid'])
```

Comment:
So did making plots based off of grid columns just not work before? Can you show me an example where this didn't work before but now, with this fix, it works?